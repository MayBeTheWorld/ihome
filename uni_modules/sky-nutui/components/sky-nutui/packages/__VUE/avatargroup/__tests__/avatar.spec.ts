import { config, mount } from '@vue/test-utils';
import AvatarGroup from '../index.vue';
import Avatar from '../../Avatar/index.vue';
import NutIcon from '../../icon/index.vue';

beforeAll(() => {
  config.global.components = {
    NutIcon
  };
});

afterAll(() => {
  config.global.components = {};
});

test('size props', async () => {
  const wrapper = mount(AvatarGroup, {
    props: {
      size: 'small'
    }
  });
  expect(wrapper.html()).toMatchSnapshot();
});
