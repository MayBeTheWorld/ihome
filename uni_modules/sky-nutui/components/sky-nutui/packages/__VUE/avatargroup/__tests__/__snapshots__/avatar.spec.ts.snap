// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`size props 1`] = `"<view class=\\"nut-avatar-group\\"></view>"`;
